CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.6.0 (2019-04-12)
+
+- Added support for headless browsers during unit and e2e tests. [#78](https://github.com/blackbaud/skyux-sdk-builder/pull/78)
+- Added TSLint rules to warn consumers when using focused or skipped tests. [#79](https://github.com/blackbaud/skyux-sdk-builder/pull/79)
```

Comment:
@Blackbaud-SteveBrush if I understood correctly skipped tests would `warn` but focused tests will `fail`. Should we reword this to something like:
` Added TSLint rules to warn consumers when using focused or skipped tests.` 

This might not make it seem like we will only `warn` for both types.